definitions/npm/react-router_v5.x.x/flow_v0.63.x-/react-router_v5.x.x.js

```diff
@@ -128,4 +128,6 @@ declare module "react-router" {
     pathname: string,
     options?: MatchPathOptions | string
   ): null | Match;
+  
+  declare export function generatePath(pattern?: string, params?: Object): string;
```

Comment:
`Object` is now an alias to `any`, here `{}` should do the work.

Reply:
✅